pages/_error.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * We rely on `newsroomContextProps` being defined in both _document.tsx and _app.tsx.
+ * This is handy if you want to show a custom error page (e.g. 404) while also rendering
+ * other components, e.g. header and/or footer with proper translations.
+ */
```

Comment:
Do we need the customer error page in the starter theme? I see that the default Next Error is rendered always, so we have no use for all the custom props.
If developers would want to see, how to enable newsroom information on the error page, they can check out the Bea repo, right?

Reply:
We need it for reasons I've mentioned in the comment otherwise the application crashes when trying to access `localeCode` in either `_document.tsx` or `_app.tsx` as `newsroomContextProps` is undefined.

We could technically try adding some conditions in those two files to render `<NextError>` but it's not exactly pretty because of the whole `this.props.__NEXT_DATA__.props.pageProps` thing 😄 

Unfortunately I wasn't able to come up with a better solution at this time.

Reply:
Oh, right! Then it all makes sense!